solr/core/src/java/org/apache/solr/schema/DenseVectorField.java

```diff
@@ -104,7 +106,7 @@ public void init(IndexSchema schema, Map<String, String> args) {
             .orElse(DEFAULT_SIMILARITY);
     args.remove(KNN_SIMILARITY_FUNCTION);
 
-    this.knnAlgorithm = args.get(KNN_ALGORITHM);
+    this.knnAlgorithm = Optional.ofNullable(args.get(KNN_ALGORITHM)).orElse(DEFAULT_KNN_ALGORITHM);
```

Comment:
use getOrDefault? https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#getOrDefault-java.lang.Object-V-

```suggestion
    this.knnAlgorithm = args.getOrDefault(KNN_ALGORITHM, DEFAULT_KNN_ALGORITHM);
```

Reply:
This should cleanup the imports too.